src/tripCard/TripCard.tsx

```diff
@@ -40,7 +38,7 @@ export type Driver = User & {
 export type TripCardProps = A11yProps &
   Readonly<{
     href: string | JSX.Element
-    itinerary: Place[]
+    itinerary: JSX.Element
```

Comment:
I'm sorry to reopen a debate, but I'm wondering if we could keep a strong typing here (to allow only Itinerary). I know it's meant to add meta tags, but as our microdata info are not about the itinerary but about the trip itself, why not have an `SEOTripCard`, instead of a SEOItinerary. What do you think?

Reply:
Or the TripCard could have a microdata props which would accept an array of meta tags.

Reply:
Take a look at the kairos PR first and we'll discuss about it after

Reply:
> Or the TripCard could have a microdata props which would accept an array of meta tags.
If we do this, it's going to be hard to change if in the future we need micro data without a `TripCard`. In both cases we're using props to render microdata inside a UI components which is not great but I would go with the smallest component unit possible which is `Itinerary`. Even though the current solution has its own drawbacks (typing), I like it more because it helps moving forward with the `NewItinerary` which is the main goal here and we can think of another scalable solution after that.